universal-application-tool-0.0.1/app/auth/saml/SamlCiviFormProfileAdapter.java

```diff
@@ -166,8 +166,10 @@ public CiviFormProfileData mergeCiviFormProfile(
   private String extractAttributeFromArrayList(SAML2Profile profile, String attr) {
     ArrayList attributeArray = profile.getAttribute(attr, ArrayList.class);
     StringJoiner sj = new StringJoiner(" ");
```

Comment:
optional: Missed it on the original CL but this can also be

```
return attributeArray.stream()
     .map(s -> (String) s)
     .collect(Collectors.joining(" "));
```

Reply:
I tried this, but the only way I could get it to work was to do:
```
    return (String) attributeArray.stream()
        .map(s -> (String) s)
        .collect(Collectors.joining(" "));
```
Idk how I feel about casting to String twice, and to me the StringJoiner looks cleaner. Thoughts?

Reply:
Don't worry about it! thanks for trying.

I'm guessing something needs coercing, maybe Collectors<String> etc, but just guessing.